### PR TITLE
Created Fast outliers function in R

### PR DESCRIPTION
**Pull request by manu2856**
Hi team,

Created the fast outliers function in R. Please review. If everything is OK , please proceed accepting th pull request.

### CODE REVIEW

File: R/fast_outliers.R [hunk 1]
```diff
@@ -4,12 +4,99 @@
...
+#' @param cols list -  List containing the columns to be analyzed.
 #' @param method string - string indicating which method to be used to identify outliers (methods available are: "Z score" or "Interquantile Range")
+#' @param threshold_low_fre double - Indicates the threshold for evaluating outliers in categorical columns.
 #'
 #' @return dataframe
 #' @export
 #' @examples
-#' fast_outliers_identifier(data = iris, method = "Z score", cols =  c("Sepal.Length", "Sepal.Width"))
-fast_outliers_id <- function(data, columns, method) {
+#' fast_outliers_identifier(data = iris, cols =  c("Sepal.Length", "Sepal.Width"), method = "z-score")
```
Comment (scao1) [inline R/fast_outliers.R]:
```suggestion
#' fast_outlier_id(data = iris, cols =  c("Sepal.Length", "Sepal.Width"), method = "z-score")
```

File: R/fast_outliers.R [hunk 2]
```diff
@@ -4,12 +4,99 @@
...
+            outlier_values = list(labels(which(score<threshold_low_freq)))
+            print(outlier_values)
+            print(table(data_no_nans[score<threshold_low_freq]))
+            outlier_count_list = append(outlier_count_list,sum(table(data_no_nans[which(score<threshold_low_freq)])))
+            outlier_values_list = append(outlier_values_list,outlier_values)
+            method_list = append(method_list,"low-freq")
+            }
+    }
+    
+    summary = tibble(
```
Comment (scao1) [inline R/fast_outliers.R]:
```suggestion
    summary = tibble::tibble(
```

File: R/fast_outliers.R [hunk 3]
```diff
@@ -4,12 +4,99 @@
...
+        no_nans = as.integer(no_nans_list),
+        perc_nans = as.double(perc_nans_list),
+        outlier_method = method_list,
+        no_outliers = outlier_count_list,
+        outlier_values = outlier_values_list
+    )
+    
+    ##Validating produced summarry contains the same number of requested columns
+
+    if (!(count(summary)[[1]]) == length(cols)){
```
Comment (scao1) [inline R/fast_outliers.R]:
```suggestion
    if (!(dplyr::count(summary)[[1]]) == length(cols)){
```

**Review by scao1 — CHANGES_REQUESTED** [review]
Please think about these changes

File: R/fast_outliers.R (hunk 3 above)
Comment (manu2856) [reply R/fast_outliers.R]: Hi @scao1 , I changed the library to "dplyr". I did a test if the function run with that one, and it did. Also when I don ?count() on jupyter lab, thats the one Im being referred to.

**Review by scao1 — APPROVED** [review]

**Pull request merged**